src/charm.py

```diff
@@ -1033,6 +1043,33 @@ def _install_deps(self) -> None:
         logger.info("Installing charm dependencies.")
         self._apt_install(["run-one"])
 
+    @staticmethod
```

Comment:
Could this be done as a function in the module rather than on the class if it doesn't require an instance?